packages/terra-application-navigation/tests/jest/tabs/TabCount.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import TabCount from '../../../src/tabs/_TabCount';
+import mockIntl from '../mockIntl';
```

Comment:
mockIntl is provided in the terra-enzyme-intl package: https://github.com/cerner/terra-enzyme-intl#mockintl

Reply:
Updated.